server/catgenome/src/main/java/com/epam/catgenome/manager/externaldb/homologene/HomologeneManager.java

```diff
@@ -127,9 +138,20 @@
         return searchResult;
     }
 
+    private void setSpeciesName(List<Gene> genes) {
+        for (Gene gene: genes) {
+            BlastTaxonomy organism = taxonomyManager.searchOrganismById(gene.getTaxId());
```

Comment:
can we maybe introduce new dao method to load from db collection of taxonomy object for the list of ids?